lua/astrocommunity/editing-support/rustown/init.lua

```diff
@@ -0,0 +1,7 @@
+return {
+  "cordx56/rustowl",
+  version = "*",
+  build = "cd rustowl && cargo install --path . --locked",
```

Comment:
I wonder if this breaks on Windows. But no idea how to fix that there.

Reply:
> I wonder if this breaks on Windows. But no idea how to fix that there.

Id be suprised, considered how simple it is. But the community can figure that out :P